edge-modules/azure-monitor/docker/manifest.yaml.template

```diff
@@ -0,0 +1,23 @@
+image: __REGISTRY__/__NAMESPACE__/azureiotedge-metrics-collector:__VERSION__
+tags: __TAGS__
+manifests:
+  -
+    image: __REGISTRY__/__NAMESPACE__/azureiotedge-metrics-collector:__VERSION__-linux-amd64
+    platform:
+      architecture: amd64
+      os: linux
+  -
+    image: __REGISTRY__/__NAMESPACE__/azureiotedge-metrics-collector:__VERSION__-linux-arm32v7
+    platform:
+      architecture: arm
+      os: linux
+  -
+    image: __REGISTRY__/__NAMESPACE__/azureiotedge-metrics-collector:__VERSION__-linux-arm64v8
+    platform:
+      architecture: arm64
+      os: linux      
+  -
+    image: __REGISTRY__/__NAMESPACE__/azureiotedge-metrics-collector:__VERSION__-windows-amd64
+    platform:
+      architecture: amd64
+      os: windows
```

Comment:
We can remove this windows manifest.

Reply:
Removed.

Reply:
@and-rewsmith @pmzara @damonbarry  We will release a Windows container version of the module, so why are we removing it?

Reply:
@veyalla We are releasing a windows container version of the module for the 1.1 branch. In the 1.1 branch all the windows related files still exist. We cannot leave the windows related files here because master/1.2 don't contain the build infra for windows.

We should iron out how we intend to release this module. The approach taken now is to couple the release of the monitor module with that of our system modules. So the next time we release 1.1 we will be releasing:
```
mcr.microsoft.com/azureiotedge-agent:1.1.0
mcr.microsoft.com/azureiotedge-hub:1.1.0
mcr.microsoft.com/azureiotedge-metrics-collector:1.1.0 (including windows)
```

And the next time we release 1.2 we will be releasing:
```
mcr.microsoft.com/azureiotedge-agent:1.2.0
mcr.microsoft.com/azureiotedge-hub:1.2.0
mcr.microsoft.com/azureiotedge-metrics-collector:1.2.0 (but not for windows, because 1.2 doesn't support windows)
```

It might be a better idea to have an independent release for this module, similar to how we release the api proxy.

Reply:
Thanks for the details @and-rewsmith. I agree we should definitely have an independent release for this module. So, we'll be releasing and servicing (.NET security patches etc.) the production image of this module from the release/1.1 branch because we will need to provide/update Windows container version of this module till 1.1 goes out of support?

Reply:
Yes I agree with @veyalla. I think we should abandon this cherry pick and keep the module code checked-in only for the 1.1 branch. Even if we eventually want to use the module in our tests for master or 1.2 branch, we can reference the tags used in that separate release process.

Reply:
Agreed. @pmzara let's close this out for now. We'll migrate this module to the main branch later when we're ready to service it from the 1.2 branch.

Reply:
@damonbarry Currently the metrics collector is versioned at 1.1.0 as it is the same release process as the system modules. I think we need to:
- Create a separate release pipeline for the metrics module (following the pattern for the api proxy)
- Decide on a versioning scheme to use (should probably just follow the api proxy one)